examples/amazon-pay-button-example/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "amazon-pay-button-example",
+  "version": "1.0.0",
+  "description": "Amazon Pay button implementation",
+  "repository": "shopware/frontends",
+  "private": true,
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/types.d.ts",
+      "import": "./dist/module.mjs",
+      "require": "./dist/module.cjs"
+    }
+  },
+  "main": "./dist/module.cjs",
+  "types": "./dist/types.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prepack": "pnpm run prepare && nuxt-module-build build",
+    "dev:playground": "pnpm run prepare && nuxi dev playground",
+    "build:playground": "pnpm run prepare && nuxi build playground",
+    "build": "pnpm run prepare && nuxt-module-build build --stub && nuxi prepare playground",
+    "lint": "pnpm run prepare && vue-tsc --noEmit",
+    "prepare": "nuxt-module-build prepare"
+  },
+  "dependencies": {
+    "@nuxt/kit": "^3.11.2"
+  },
+  "devDependencies": {
+    "@amazonpay/amazon-pay-api-sdk-nodejs": "^2.3.1",
+    "@nuxt/devtools": "^1.3.2",
+    "@nuxt/module-builder": "^0.7.0",
+    "@nuxt/schema": "^3.11.2",
+    "@shopware-pwa/composables-next": "canary",
+    "@shopware/api-client": "canary",
+    "@types/node": "^20.14.1",
+    "nuxt": "^3.11.2",
+    "typescript": "latest",
+    "vitest": "^1.6.0",
+    "vue-tsc": "^2.0.19"
```

Comment:
please make all dependencies strict and make sure versions are the same with other versions in repo